.travis.yml

```diff
@@ -2,6 +2,8 @@ rvm:
   - 2.1.9
   - 2.2.4
```

Comment:
It may make sense to remove these EOL versions too?

Reply:
Yes, I think it does.  I just wasn't sure if we wanted to leave those in for now, and was planning to make another PR to take them out once we'd established that the newer versions of ruby worked.

I need to make an update to the README as well, once we know these are working.

Reply:
readme is updated.

Do you think we should remove the old versions of ruby now, or leave them in place for a bit in case there are any bugs we're not aware of?

Reply:
Excellent!

No idea about removing the EOL versions. Maybe a project maintainer will give good advice in respect to this :wink: